app/src/org/commcare/android/database/user/models/SessionStateDescriptor.java

```diff
@@ -91,7 +91,7 @@ private String createSessionDescriptor(CommCareSession session) {
         //TODO: Serialize into something more useful. I dunno. JSON/XML/Something
         StringBuilder descriptor = new StringBuilder();
         for (StackFrameStep step : session.getFrame().getSteps()) {
-            descriptor.append(step.getType());
+            descriptor.append(step.getType()).append(" ");
             if (SessionFrame.STATE_COMMAND_ID.equals(step.getType())) {
                 descriptor.append(step.getId());
```

Comment:
I'm actually a little confused because I moved the space addition to the bottom of the loop body in https://github.com/dimagi/commcare-android/pull/1059/files#diff-cd3d036d05167f3e910a1381ea329b91R106